apps/fm/test/unit/fm_test.js

```diff
@@ -338,6 +352,20 @@ suite('FM', function() {
       test('#airplane-mode-warning is hidden', function() {
         assert.equal(!!$('airplane-mode-warning').hidden, true);
       });
+
+      test('#antenna-warning is on and container is hidden', function() {
```

Comment:
Lets combine these 2 tests into 1. We also need a similar one for the airplane mode as well as the permutation of both: e.g. have antenna but in airplane mode, not in airplane mode but do not have an antenna.